.gitignore

```diff
@@ -389,6 +389,7 @@ Makefile.am-stamp
 /maint/createhtmlindex
 /maint/extractcvars
 /maint/cvardirs
+/maint/f77tof90
```

Comment:
Are you seeing this file generated? It should be moved to `test/mpi/maint`.

Reply:
@yfguo - I think this just comes from working on a branch from before the patch that moved this and then changing to a branch from after. I see this a lot.